frontend/src/modules/Environments/components/EnvironmentTeams.js

```diff
@@ -325,6 +332,31 @@ export const EnvironmentTeams = ({ environment }) => {
     }
   };
 
+  const updateConsumptionRoleHandler = async (newRow) => {
+    const response = await client.mutate(
+      updateConsumptionRole({
+        environmentUri: environment.environmentUri,
+        consumptionRoleUri: newRow.consumptionRoleUri,
+        input: {
+          groupUri: newRow.groupUri,
+          consumptionRoleName: newRow.consumptionRoleName
+        }
+      })
+    );
+    if (!response.errors) {
+      enqueueSnackbar('Consumption Role was updated', {
+        anchorOrigin: {
+          horizontal: 'right',
+          vertical: 'top'
+        },
+        variant: 'success'
+      });
+      fetchRoles();
+    } else {
+      throw new Error(response.errors[0].message);
```

Comment:
Is there a reason we throw Error here instead of calling `dispatch(SET_ERROR ...)` ?

Reply:
That's correct, `onProcessRowUpdateError` will call dispatch for everything that goes wrong